internal/cmd/metrics.go

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package cmd
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	var cmd = &cobra.Command{
+		Use:          "metrics",
+		Short:        "Optain the CPU/memory usage stats for a given service",
```

Comment:
```suggestion
		Short:        "Outputs the CPU/memory usage stats for a given EdgeX microservice",
```

Reply:
I changed this to  `Outputs the CPU/memory usage stats for EdgeX microservices`